# Use system binaries for those that are not easy to build on Windows instead of Chromium's binaries

--- a/third_party/WebKit/Source/build/scripts/scripts.gypi
+++ b/third_party/WebKit/Source/build/scripts/scripts.gypi
@@ -49,8 +49,8 @@
         ],
         'conditions': [
             ['OS=="win"', {
-                'gperf_exe': '<(DEPTH)/third_party/gperf/bin/gperf.exe',
-                'bison_exe': '<(DEPTH)/third_party/bison/bin/bison.exe',
+                'gperf_exe': 'gperf',
+                'bison_exe': 'bison',
               },{
                 'gperf_exe': 'gperf',
                 'bison_exe': 'bison',
